docs-generation/main.py

```
@@ -221,16 +225,23 @@ def prepare_translation(self, lang: str, mode: str, count: int = None, force_all
             f"Preparing {lang} translation batch ({mode} mode{' + force-all-nodes' if force_all_nodes else ''})"
         )
     
-    def translate_docs(self, lang: str, mode: str, count: int = None, force: bool = False) -> bool:
+    def translate_docs(self, lang: str, mode: str, count: int = None, force: bool = False, node_name: str = None, concurrency: int = 1) -> bool:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)